openwisp-config/files/openwisp.agent

```diff
@@ -193,6 +193,7 @@ APPLYING_CONF="$WORKING_DIR/applying_conf"
 BOOTUP="$WORKING_DIR/bootup"
 REGISTRATION_URL="$URL/controller/register/"
 UNMANAGED_DIR="$WORKING_DIR/unmanaged"
+TEMP_ERROR_LOG="$WORKING_DIR/error-log.tmp"
```

Comment:
```suggestion
CONFIG_ERROR_LOG="$WORKING_DIR/config-error.log"
```

Anything in `/tmp/openwisp` is temporary (that's why it's called `/tmp`) so we don't need to repeat it in the name.